rekall-core/rekall/plugins/addrspaces/vmi.py

```diff
@@ -1,4 +1,9 @@
-from libvmi import Libvmi
+try:
```

Comment:
How about the simpler:
```
libvmi = None
try:
   from libvmi import libvmi
except ImportError: pass
```


Alternatively we can just conditionally import this entire file in the\_\_init\_\_.py file